src/client/wire_protocol/operations.rs

```diff
@@ -306,6 +318,12 @@ impl Message {
         match self {
             /// Only the server should sent replies
             &Message::OpReply {..} => Err("OP_REPLY should not be sent by the client".to_owned()),
+            &Message::OpInsert {
+                header: ref h,
+                flags: ref f,
+                full_collection_name: ref fcn,
```

Comment:
ditto here, write out 'full_collection_name' instead of 'fcn', 'header' instead of 'h', etc.